### PR TITLE
Aerospike vector store bedrock example for  AWS sample repo

### PR DESCRIPTION
**Pull request by rahul-aerospike**

### CODE REVIEW

File: aerospike/samples/avs_bedrock_integration.ipynb [hunk 1]
```diff
@@ -0,0 +1,400 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "IQt5lMKvxios"
+      },
+      "source": [
+        "# [Aerospike Vector Store(AVS)](https://aerospike.com/docs/vector) demo with [Amazon Bedrock](https://aws.amazon.com/bedrock/)\n",
```
Comment (arrowplum) [inline aerospike/samples/avs_bedrock_integration.ipynb:9]: We aren't here really explaining what this demo is going to show (namely an example of switching models) Are we still seeing that as the main advantage with bedrock (+aerospike)?

File: aerospike/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Aerospike
+## Aerospike Vector Search (AVS)
+
+This directory contains pre-built samples to help customers get started with the **Generative AI** using **AWS** services and **Aerospike Vector Search**.[AVS](https://aerospike.com/docs/vector).
+
+## Get Started
+
+### Prerequisites
+
+Follow these [steps](https://aerospike.com/docs/vector/quickstart) to setup Aerospike Vector Store cluster.
```
Comment (hev) [inline aerospike/README.md]: This link is actually takes you to a place that assumes you have things installed. 
```suggestion
Follow these [steps](https://aerospike.com/docs/vector/install) to setup Aerospike Vector Store cluster.
```
Update to install specific instructions.

**Review by hev — CHANGES_REQUESTED** [review]
Small link improvement but overall I think we should merge and submit into the actually community repo.

**Review by hev — APPROVED** [review]

File: aerospike/samples/avs_bedrock_integration.ipynb (hunk 1 above)
Comment (rahul-aerospike) [reply aerospike/samples/avs_bedrock_integration.ipynb:9]: The notebook, on the top states that we are analysing text using mentioned amazon text models. It also states that it uses AVS for vector storage and retrieval. What other information you think should be added? @arrowplum .

File: aerospike/samples/avs_bedrock_integration.ipynb [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+        "\n",
+        "- In this notebook, we use two Amazon Titan text models—`amazon.titan-embed-text-v2:0` and `amazon.titan-embed-text-v1` to analyze text.\n",
```
Comment (arrowplum) [inline aerospike/samples/avs_bedrock_integration.ipynb]:
```suggestion
        "- In this notebook, we use Amazon Bedrock to easily compare results from two similar models used to analyze text.\n",
```

**Review by arrowplum — APPROVED** [review]
Small suggestion. Otherwise approve.

**Pull request merged**